cmd/expiration-mailer/main.go

```diff
@@ -456,6 +456,10 @@ func (m *mailer) sendToOneRegID(ctx context.Context, conn bmail.Conn, regID int6
 		var badAddrErr *bmail.BadAddressSMTPError
 		if errors.Is(err, errNoValidEmail) || errors.As(err, &badAddrErr) {
 			m.updateLastNagTimestamps(ctx, parsedCerts)
+			// Though these are technically an error, we don't consider them
+			// failures to send because these (lack of) addresses were not even
+			// attempted to be sent to.
```

Comment:
This sentence is tied in knots by the passive voice[1].

```suggestion
			// Though these are technically an error, we don't consider them
			// failures to send because we did not even attempt to send anything.
```

Though writing that I realize it's inaccurate for the `BadAddressSMTPError`, which is an error we receive from the remote server.

```suggestion
			// Some accounts have no email; some accounts have a nonexistent
			// email. Treat those as non-error cases.
```

[1]: Which is not always bad! See [1].